internal/pkg/v2/controller/errorconcept/default.go

```diff
@@ -0,0 +1,110 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package errorconcept
```

Comment:
Why name `errorconcept`? what does `concept` mean in this domain? These are custom errors, so why not just `error` or `errors`

Reply:
The `errorconcept` package is inherited from v1 [internal/pkg/errorconcept/handler.go](https://github.com/edgexfoundry/edgex-go/blob/master/internal/pkg/errorconcept/handler.go), and we'll be happy to rename the package name if needed.